sdk/js/src/Request.ts

```diff
@@ -24,36 +24,24 @@ export const EVENT_URL = 'https://events'
 
 export const CONFIG_PATH = '/v1/sdkConfig'
 export const EVENTS_PATH = '/v1/events'
+export const SAVE_ENTITY_PATH = '/v1/entities'
 
-export const baseRequestHeaders = (environmentKey?: string): AxiosRequestHeaders => {
+export const baseRequestHeaders = (environmentKey?: string, enableCloudData?: boolean): AxiosRequestHeaders => {
     return {
         'Content-Type': 'application/json',
-        ...(environmentKey ? { 'Authorization': environmentKey } : {})
+        ...(environmentKey ? { 'Authorization': environmentKey } : {}),
+        ...(enableCloudData ? {'Enable-Cloud-Entity-Data': `${enableCloudData}`} : {})
```

Comment:
what's the header for? isnt the API just expecting a query param?

Reply:
this is off of rabia's PR since i made it off of her pr

Reply:
ah, looks like you need to rebase then